src/main/java/jp/classmethod/aws/gradle/cloudformation/AmazonCloudFormationMigrateStackTask.java

```diff
@@ -163,8 +163,8 @@ private void updateStack(AmazonCloudFormation cfn) throws IOException {
 		}
 		if (isCapabilityIam()) {
 			Capability selectedCapability =
-					(useCapabilityIam == null) ? Capability.CAPABILITY_IAM : useCapabilityIam;
-			getLogger().error("Using policy: " + selectedCapability);
+					(getUseCapabilityIam() == null) ? Capability.CAPABILITY_IAM : getUseCapabilityIam();
+			getLogger().info("Using IAM capability: " + selectedCapability);
```

Comment:
Unsure what the right way to go about resolving merge issues in the github context. I have another change: 920da08

IT looks like it's listed on this pull request.